build failure on FreeBSD (depext-1.0.3)

```
#=== ERROR while installing depext.1.0.3 ======================================#
# opam-version 1.2.2
# os           freebsd
# command      gmake
# path         /home/hannes/.opam/m404/build/depext.1.0.3
# compiler     4.04.0
# exit-code    2
# env-file     /home/hannes/.opam/m404/build/depext.1.0.3/depext-5744-b1c0e4.env
# stdout-file  /home/hannes/.opam/m404/build/depext.1.0.3/depext-5744-b1c0e4.out
# stderr-file  /home/hannes/.opam/m404/build/depext.1.0.3/depext-5744-b1c0e4.err
### stdout ###
# [...]
# chmod a+x shell/build.sh
# cd src_ext && gmake
# gmake[1]: Entering directory '/usr/home/hannes/.opam/m404/build/depext.1.0.3/src_ext'
# [ -e  cmdliner-0.9.8.tbz ] || \
# curl -OL http://erratique.ch/software/cmdliner/releases/cmdliner-0.9.8.tbz && ocaml ../shell/md5check.ml cmdliner-0.9.8.tbz fc67c937447cc223722f1419fa2189da || \
# { curl -OL https://opam.ocaml.org/2.0~dev/cache/md5/fc/fc67c937447cc223722f1419fa2189da && mv fc67c937447cc223722f1419fa2189da cmdliner-0.9.8.tbz && ocaml ../shell/md5check.ml cmdliner-0.9.8.tbz fc67c937447cc223722f1419fa2189da; }
# cmdliner-0.9.8.tbz has the expected MD5.
# mkdir -p tmp
# cd tmp && bunzip2 -c ../cmdliner-0.9.8.tbz | tar x
# gmake[1]: Leaving directory '/usr/home/hannes/.opam/m404/build/depext.1.0.3/src_ext'
### stderr ###
# tar: Error opening archive: Failed to open '/dev/sa0'
# gmake[1]: *** [Makefile:49: cmdliner.stamp] Error 1
# gmake: *** [Makefile:2: all] Error 2
```

--- a/src_ext/Makefile
+++ b/src_ext/Makefile
@@ -46,7 +46,7 @@ endef
 
 %.stamp: %.download
 	mkdir -p tmp
-	cd tmp && $(if $(patsubst %.tar.gz,,$(URL_$*)),bunzip2,gunzip) -c ../$(notdir $(URL_$*)) | tar x
+	cd tmp && $(if $(patsubst %.tar.gz,,$(URL_$*)),bunzip2,gunzip) -c ../$(notdir $(URL_$*)) | tar xf -
 	rm -rf $*
 	@for ii in tmp/*; do if [ -d $${ii} ]; then mv $${ii} $*; fi; done; \
 	rm -rf tmp

